src/components/Dashboard/Form/Applicants/Applicants.js

```diff
@@ -1,65 +1,73 @@
-import React from "react";
+import React, { useState } from "react";
 import FieldWrapper from "../FieldWrapper/FieldWrapper";
 import styles from "./Applicants.scss";
 
-import { reasonsList, disabilityList } from "../../../../staticData";
+import { reasonsList } from "../../../../staticData";
 
-const Applicants = () => (
-  <div className={`FormContent ${styles.Applicants}`}>
-    <FieldWrapper
-      name={`applicant.names`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.names`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.city`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.postalCode`}
-      componentSize="medium"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.street`}
-      componentSize="large"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.houseNumber`}
-      componentSize="medium"
-      component="input"
-    />
-    <FieldWrapper
-      name={`applicant.issue`}
-      componentSize="large"
-      component="select"
-      options={reasonsList}
-    />
-    <FieldWrapper
-      name={`applicant.reason`}
-      componentSize="extraLarge"
-      component="select"
-      options={disabilityList}
-    />
-    <FieldWrapper
-      name={`applicant.secondReason`}
-      componentSize="extraLarge"
-      component="select"
-      options={disabilityList}
-    />
-    <FieldWrapper
-      name={`applicant.period`}
-      componentSize="medium"
-      component="input"
-    />
-  </div>
-);
+const Applicants = () => {
+  const [chosenParent, setChosenParent] = useState("WYBIERZ_WNIOSEK");
+
+  const getOptions = () =>
+    reasonsList.find(el => el.value === chosenParent).disabilityListItems;
+
+  return (
+    <div className={`FormContent ${styles.Applicants}`}>
+      <FieldWrapper
+        name={`applicant.names`}
+        componentSize="large"
+        component="input"
+      />
+      <FieldWrapper
+        name={`applicant.names`}
+        componentSize="large"
+        component="input"
+      />
+      <FieldWrapper
+        name={`applicant.city`}
+        componentSize="large"
+        component="input"
+      />
+      <FieldWrapper
+        name={`applicant.postalCode`}
+        componentSize="medium"
+        component="input"
+      />
+      <FieldWrapper
+        name={`applicant.street`}
+        componentSize="large"
+        component="input"
+      />
+      <FieldWrapper
+        name={`applicant.houseNumber`}
+        componentSize="medium"
+        component="input"
+      />
+      <FieldWrapper
+        name={`applicant.issue`}
+        componentSize="large"
+        component="select"
+        options={reasonsList}
+        onChange={e => setChosenParent(e.target.value)}
```

Comment:
I know that `e` means `event` in this context. But maybe you can use full word (`event`) instead of `e`. The latter is widely used in the context of `error`. Just to improrve readability.